opencog/openpsi/utilities.scm

```diff
@@ -326,13 +326,14 @@
 		result
 		; else check if entity is an evaluation or predicate or schema
 		(let ((type (cog-type entity)))
-			(if (or (equal? type 'PredicateNode)
-				   (equal? type 'GroundedPredicateNode)
+			(if (or (equal? type 'GroundedPredicateNode)
 				   (equal? type 'DefinedPredicateNode))
 				(set! result (cog-evaluate! (Evaluation entity (List)))))
-			(if (or (equal? type 'SchemaNode)
+			(if (equal? type 'PredicateNode)
+				(set! result (cog-tv entity)))
+			(if (or ; (equal? type 'SchemaNode) ;FIXME SchemaNode not executable
```

Comment:
@eddiemonroe how are you using `SchemaNode`? I commented it out as they are not handled by `ExecutableOutputLink`.